### ISSUE TITLE
Group Verification and Publishing of Activities

### ISSUE THREAD

**Issue by amy-silcock** [younginnovations/aidstream#119]
Hi AidStream,

A few organisations have been asking if it would be possible to add a check list to the activities on AidStream. So that they could select multiple to verify/complete/publish at the same time.

This is particulary for organisation who are publishing a large number of activities. Do let me know.

Thank you,

Amy

**Comment by sachit-singh** [comment]
Yes, this is completely possible. I will let you know once this feature has been released.
